docker/data_source_docker_plugin.go

```diff
@@ -0,0 +1,75 @@
+package docker
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceDockerPlugin() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceDockerPluginRead,
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"alias": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Docker Plugin alias.",
+			},
+
+			"plugin_reference": {
+				Type:        schema.TypeString,
+				Description: "Docker Plugin Reference.",
+				Optional:    true,
+			},
+			"disabled": {
```

Comment:
do you think this should be changed to 'enabled'?
A default should also be set for this attribute.

Reply:
Fixed. 5f925d659961469a11fa1c16c474475ad15f17a9